src/lib/lazy-load.js

```diff
@@ -1,18 +1,72 @@
 const lazyLoadingModifier = 'lazy-loading';
 const lazyLoadingImageClass = `n-image--${lazyLoadingModifier}`;
 const lazyLoadingWrapperClass = `n-image-wrapper--${lazyLoadingModifier}`;
+const performanceSupported = ('performance' in window && typeof window.performance.mark === 'function');
```

Comment:
Yeah but how do I use it from here?  Spent hours trying to figure that out before giving up (also why I'm not using `broadcast` from utils